steering_vectors/steering_vector.py

```diff
@@ -1,6 +1,6 @@
 from contextlib import contextmanager
 from dataclasses import dataclass, replace
-from typing import Any, Callable, Generator, Optional, overload
+from typing import Any, Callable, Generator, List, Optional, overload
```

Comment:
nit: no need to use `List`, we can just use `list` since we're targeting Python 3.10+

Reply:
Done in [83b0ccf](https://github.com/steering-vectors/steering-vectors/pull/21/commits/83b0ccff5fa9fdedafe08830592102e55fd0b980)